internal/services/types.go

```diff
@@ -0,0 +1,28 @@
+package services
+
+type ServiceResponse struct {
+	Service *Service `json:"service"`
+}
+type Service struct {
+	UpdatedAt              string `json:"updated_at"`
+	AlternateName          string `json:"alternate_name"`
```

Comment:
My comments on the SQL null thing are making me wonder about the nullability of fields in this struct. Does the way you've declared these types allow any of these to be null? What happens if the field is null in the database when you unmarshal into this struct?

Also, if it's the case that this code will return the empty string when in encounters a null in the database, I'm not 100% sure if that is drop-in equivalent for our frontend code. We fortunately converted all the frontend code to TypeScript, and we explicitly have `nulls` in the frontend's version of this struct (https://github.com/ShelterTechSF/askdarcel-web/blob/70664042421e2278e34d9b2c2822c827fb625eeb/app/models/Service.ts#L20-L52). In JavaScript, the empty string, `null`, and `undefined` will fortunately evaluate to `false` inside of conditional expressions, but we'd have to double check that we're not doing explicit equality checks against `null` anywhere in the frontend.

Reply:
I will match this so these can return null in the same manner we do currently, if we want to tweak this in the future we can but not a problem to change for now.

Reply:
added a commit that makes this adjustment